core/config/src/main/java/org/apache/causeway/core/config/CausewayConfiguration.java

```diff
@@ -2085,6 +2085,19 @@ public static class Po {
                     Mode mode = Mode.WRITE;
                 }
             }
+
+            private final EntityPropertyChangePublisher entityPropertyChangePublisher = new EntityPropertyChangePublisher();
+
+            @Data
+            public static class EntityPropertyChangePublisher {
+
+                private final Bulk bulk = new Bulk();
+
+                @Data
+                public static class Bulk {
+                    int threshold = 1;
```

Comment:
We need an integration test that demonstrates when the value of this is changed from its default.

Reply:
Didn’t know how strict your testing was. Apparently it’s quite strict.

Reply:
Added regression test with different causeway property.